### DISCUSSION TITLE
Throwing this into `bt` or `backtesting.py` to accelerate backtesting?

### DISCUSSION THREAD

**Discussion by BradKML** [Heerozh/spectre#26] [Ideas]
Currently there are a few well-established backtesters out there that are not necessarily optimized for large-sized backtesting, but his repo seems promising with GPU acceleration. Ditto for technical analysis libraries that runs on pandas instead of optimized on a lower level.

**Comment by Heerozh** [comment]
this project mainly pursues performance.
squeeze performance often requires redesigning the architecture,
design acceleration for other project has limited effect

**Reply by BradKML** [reply]
I think the projects in question are also hoping for performance increases by shaking up their own architecture due to slowing development. They do however cover a broader range of backtesting techniques and have established userbase.

**Reply by Heerozh** [reply]
highly impossible, gpu acceleration has strict restrictions, and all algorithms need to be vectorized, vram is also very limited.

u can use spectre to generate factors, and then use other backtesting libraries to perform backtests based on spectre results, which can take advantage of gpu acceleration.
